src/results/proforma.jl

```diff
@@ -120,19 +107,95 @@ function proforma_results(p::REoptInputs, d::Dict)
             year_one_fuel_cost_bau = d["Generator"]["year_one_fuel_cost_before_tax_bau"]
         end
         if !third_party
-            annual_om = -1 * (fixed_and_var_om + d["Generator"]["year_one_fuel_cost_before_tax"])
+            annual_fuel = -1 * d["Generator"]["year_one_fuel_cost_before_tax"]
+            annual_om = -1 * fixed_and_var_om
 
-            annual_om_bau = -1 * (fixed_and_var_om_bau + year_one_fuel_cost_bau)
+            annual_fuel_bau = -1 * year_one_fuel_cost_bau
+            annual_om_bau = -1 * fixed_and_var_om_bau
         else
+            annual_fuel = 0.0
             annual_om = -1 * fixed_and_var_om
 
+            annual_fuel_bau = 0.0
             annual_om_bau = -1 * fixed_and_var_om_bau
         end
 
+        m.om_series += escalate_fuel(annual_fuel, p.s.financial.generator_fuel_cost_escalation_rate_fraction)
+        m.om_series += escalate_om(annual_om)
+        m.om_series_bau += escalate_fuel(annual_fuel_bau, p.s.financial.generator_fuel_cost_escalation_rate_fraction)
+        m.om_series_bau += escalate_om(annual_om_bau)
+    end
+
+    # calculate CHP o+m costs, incentives, and depreciation
+    if "CHP" in keys(d) && d["CHP"]["size_kw"] > 0
+        update_metrics(m, p, p.s.chp, "CHP", d, third_party)
+    end
+
+    # calculate (new) Boiler o+m costs (just fuel, no non-fuel operating costs currently)
+    # the optional installed_cost inputs assume net present cost so no option for MACRS or incentives
+    if "ExistingBoiler" in keys(d) && d["ExistingBoiler"]["size_mmbtu_per_hour"] > 0
+        fuel_cost = d["ExistingBoiler"]["year_one_fuel_cost_before_tax"]
+        m.om_series += escalate_fuel(-1 * fuel_cost, p.s.financial.existing_boiler_fuel_cost_escalation_rate_fraction)
+        var_om = 0.0
+        fixed_om = 0.0
+        annual_om = -1 * (var_om + fixed_om)
         m.om_series += escalate_om(annual_om)
+        
+        # BAU ExistingBoiler
+        fuel_cost_bau = d["ExistingBoiler"]["year_one_fuel_cost_before_tax_bau"]
+        m.om_series_bau += escalate_fuel(-1 * fuel_cost_bau, p.s.financial.existing_boiler_fuel_cost_escalation_rate_fraction)
+        var_om_bau = 0.0
+        fixed_om_bau = 0.0
+        annual_om_bau = -1 * (var_om_bau + fixed_om_bau)
         m.om_series_bau += escalate_om(annual_om_bau)
+    end    
+
+    # calculate (new) Boiler o+m costs and depreciation (no incentives currently)
```

Comment:
Should instead say (no incentives currently, other than MACRS)?

Reply:
Done